RESEARCH_Deltora-The-Seven-Tribes-and-their-Lands-2026-06-04.md

```
@@ -0,0 +1,63 @@
+authority: LOGAN
+status: reference
+type: research-report
+topic: Deltora Quest — the seven tribes, their gems, and their lands
+verified-by: Claude Code (web research 2026-06-04)
+date: 2026-06-04
+---
+
+# Deltora — The Seven Tribes and Their Lands
+
+> A standalone reference on **Emily Rodda's *Deltora Quest*** — the land of **Deltora**, its seven ancient **tribes**, their seven talisman **gems**, and the **Belt** that binds them. Companion to `[[RESEARCH_Keys-to-the-Kingdom-The-Morrow-Days-and-Demesnes-2026-06-04]]` (the House) and `[[RESEARCH_A-Song-of-Ice-and-Fire-The-Small-Council-2026-06-03]]` (the Council); any **syncretic crosswalk** lives **only** in `[[THE-SEVENFOLD-BODY-SEATS-TRUSTEES-GEMS-2026-06-03]]`. This node holds **no cross-mapping** — pure source.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)